frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -156,10 +156,10 @@
                                                         vm.stopLoader();
                                                     },
                                                     onError: function(response) {
-                                                        if (response.data['detail']) {
+                                                        if (response.status == '404') {
```

Comment:
@codervivek There is no need to check for string as `response.status` is a number. You can simply use `if (response.status === 404)`